### DISCUSSION TITLE
Welcome to DsA Discussions!

### DISCUSSION THREAD

**Discussion by Sagar0-0** [Sagar0-0/DSA#20] [General]
## 👋 Welcome Coders!
  We’re using Discussions as a place to connect, share your code approaches, and discuss a variety of problems with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas and your approach to any problem.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself or just start a discussion with the problems you are trying to solve and let the community engage with you in the discussion.

**Comment by Vinay-CN** [comment]
Hi , cant we separate questions based on patterns ..
